FrontEnd/Hopsy/src/screens/HomeScreen.js

```diff
@@ -64,9 +27,8 @@ export default class Test extends React.Component {
             <Icon name="ios-heart-empty" size={24} color="#737888" style={{marginRight: 16}}/>
             <Icon name="ios-chatboxes" size={24} color="#737888" style={{marginRight: 16}}/>
```

Comment:
Maybe put this in a style sheet.

Reply:
Since the styles are so short I did not create a separate style sheet for them